Quadriga/src/main/webapp/WEB-INF/views/transformation/transformation.jsp

```diff
@@ -0,0 +1,226 @@
+		});
+		
+	});
+	
+</script>
+<script type="text/javascript">
+$(document).ready(function () {
+    $('#confirmationTransformation').click(function () {
+    	if(jQuery('#divProjectList input[type=checkbox]:checked').length && jQuery('#headingTwo input[type=checkbox]:checked').length) {
+        	var projects = [];
+            $.each($("input[name='project']:checked"), function(){            
+                projects.push($(this).val());
+            });        
+    		var transformations = [];
+            $.each($("input[name='transformation']:checked"), function(){            
+                transformations.push($(this).val());
+            });
+        	$('#confirm').dialog({ modal: true,
+        						draggable:false,
+        						resizable:false,
+       							position:'center',
+       							width:'auto',
+       							height:'auto',
+       							title:"Confirm transformation",
+        	   							
+       							open: function (type, data) {
+       						        $(this).parent().appendTo("form");
+       						    },
+       						    
+	       						 buttons: { "Submit": function() { $(this).dialog("close"); },"Cancel": function() { $(this).dialog("close"); } },
+       							
+				        		open: function(){           
+				        			var projects = [];
+				        	        $.each($("input[name='project']:checked"), function(){            
+				        	            projects.push($(this).val());
+				        	        });
+				        			
+				        			var transformations = [];
+				        	        $.each($("input[name='transformation']:checked"), function(){            
+				        	            transformations.push($(this).val());
+				        	        });
+    			
+				        			jQuery("#contentholder").html("<h3>Transformations:</h3> "+transformations.join("<br/>")+""+" <br/> <br/><h3>Projects </h3> " + projects.join("<br/>")+"");	
+				        		}  		
+				        	  }
+        					);   
+    					}
+		    	
+    				else{
+				 		$('#alert').dialog({ modal: true,
+						draggable:false,
+						resizable:false,
+						position:'center',
+						width:'auto',
+						height:'auto',
+						title:"Transformation cannot be processed. ",
+									
+						open: function (type, data) {
+						    $(this).parent().appendTo("form");
+						    },
+						 buttons: { "Okay": function() { $(this).dialog("close"); } },
+		    				});	
+						jQuery("#alertholder").html("Please select at least one transformation and project");
+		    			}
+    });    
+});
+</script>
+
+<style>
+ #project div {
+  display: inline;
+  margin: 0 1em 0 1em;
+  width: 30%;
+}
+</style>
+
+
+<div>
+<header>
+	<h2><label><input type="checkbox" id="selectAllProjects"></label> Projects</h2>
+	</header>
+</div>
+
+<c:choose>
+<c:when test="${show_error_alert}">
+<div class="alert alert-danger">
+<button type="button" class="close" data-dismiss="alert">&times;</button>
+    ${error_alert_msg}
+</div>
+</c:when>
+</c:choose>
```

Comment:
You don't need this. The code to display the error message is already in the skeleton.